package.json

```diff
@@ -63,7 +63,7 @@
     "prettier": "^3.0.0"
   },
   "engines": {
-    "node": "^14.16.0 || >=16.0.0"
+    "node": ">=18.20.0"
```

Comment:
We announced dropping support for 14 and 16. Doesn't this mean we'll be dropping more than we announced?

Reply:
As far as I know were always aiming for at least `18.14`, from there the eventual bump to `18.20` was based on needing to support specific functionality outlined in the description of this change. Open to discussing further and to collaborating on the best way forward.